### PR TITLE
Add section on downloading NeMo datasets to USER_GUIDE.md

### PR DESCRIPTION
**Pull request by TaekyungHeo**
## Summary
Add section on downloading NeMo datasets to USER_GUIDE.md

## Test Plan
Not needed

### CODE REVIEW

File: USER_GUIDE.md [hunk 1]
```diff
@@ -195,3 +197,44 @@ cloudai --mode generate-report \
...
+You should have at least 1TB space in your data directory (data_dir) as the Pile dataset size is 800GB. Alternatively, you can control the file numbers to download: link
+
+Please make sure to clean up your data_dir clean before submitting the job:
+```bash
+$ rm -rf /mswg2/E2E/theo/*
+```
+
+You can download the datasets with the following command:
+```bash
+$ python3 launcher_scripts/main.py \
```
Comment (amaslenn) [inline USER_GUIDE.md:213]: Should we suggest using the same python we use for Cloud AI CLI? And that suggests, that we need to add a section that describes how to initialise it like we have in README.

File: USER_GUIDE.md [hunk 2]
```diff
@@ -195,3 +197,44 @@ cloudai --mode generate-report \
...
 `--output-dir` accepts one scenario run results directory.
+
+## Installing the NeMo Datasets (Pile or mC4)
+Basically, you can install the NeMo launcher test template by following the README file of CloudAI. However, in addition to the steps, you need to download NeMo datasets manually. To understand datasets available in the NeMo framework, you can take a look at the Data Preparation section of [the document](https://docs.nvidia.com/nemo-framework/user-guide/latest/llms/baichuan2/dataprep.html). According to the document, NeMo supports two types of datasets depending on the model that you want to train: The Pile and mC4. It also provides detailed instructions on how to download these datasets for various platforms. Let’s assume that we have a slurm cluster.
+
+You should have at least 1TB space in your data directory (data_dir) as the Pile dataset size is 800GB. Alternatively, you can control the file numbers to download: link
+
+Please make sure to clean up your data_dir clean before submitting the job:
+```bash
+$ rm -rf /mswg2/E2E/theo/*
```
Comment (amaslenn) [inline USER_GUIDE.md:208]: What is `data_dir` and why do we have this hardcoded path here?

File: USER_GUIDE.md [hunk 3]
```diff
@@ -195,3 +197,44 @@ cloudai --mode generate-report \
 ```
 
 `--output-dir` accepts one scenario run results directory.
+
+## Installing the NeMo Datasets (Pile or mC4)
+Basically, you can install the NeMo launcher test template by following the README file of CloudAI. However, in addition to the steps, you need to download NeMo datasets manually. To understand datasets available in the NeMo framework, you can take a look at the Data Preparation section of [the document](https://docs.nvidia.com/nemo-framework/user-guide/latest/llms/baichuan2/dataprep.html). According to the document, NeMo supports two types of datasets depending on the model that you want to train: The Pile and mC4. It also provides detailed instructions on how to download these datasets for various platforms. Let’s assume that we have a slurm cluster.
+
+You should have at least 1TB space in your data directory (data_dir) as the Pile dataset size is 800GB. Alternatively, you can control the file numbers to download: link
```
Comment (amaslenn) [inline USER_GUIDE.md:204]: I'm sure you know, but to highlight that the link is missing.

File: USER_GUIDE.md [hunk 4]
```diff
@@ -195,3 +197,44 @@ cloudai --mode generate-report \
...
+    data_preparation.run.time_limit="24:00:00"
+```
+
+Additional configurations that are available for adjustment can be found in the following configuration files:
+- `launcher_scripts/conf/config.yaml`
+- `launcher_scripts/conf/data_preparation/gpt3/download_gpt3_pile.yaml`
+
+When you execute the `squeue` command, you can find the list of jobs that you submitted:
+```bash
+$ squeue -u theo
```
Comment (amaslenn) [inline USER_GUIDE.md:229]: Let's sanitise the output to avoid names for users and cluster.

File: USER_GUIDE.md (hunk 1 above)
Comment (TaekyungHeo) [reply USER_GUIDE.md:209]: CloudAI CLI cannot download the datasets. It is not CloudAI's responsibility. Please review the updated USER_GUIDE.md.

**Review by srinivas212 — APPROVED** [review]

**Pull request merged**